realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -327,6 +335,15 @@ public Builder deleteRealmIfMigrationNeeded() {
         }
 
         /**
+         * Settings this will create a in-memory Realm instance.
```

Comment:
Yes, checked relevant doc from IOS, and added some notes to `inMemory()` in the 2nd commit. Please help to check if it is clear enough.